src/huutopussi_server/model.clj

```diff
@@ -146,14 +146,15 @@
   ;TODO Works only with exactly 2 teams with both having 2 players
   (let [[team1-first-player team1-second-player team2-first-player team2-second-player] (mapcat val teams)
         player-ids [team1-first-player team2-first-player team1-second-player team2-second-player]
+        sort-hand (fn [cards] (reverse (sort-by (juxt :suit :value) cards)))
```

Comment:
Better place for this sorting logic is `deck` namespace. `model` is a namespace for the essential game rules. This sorting is a nice-to-have feature for users so that is easier for them to play the game.

Reply:
Now it is finally moved to `deck` ns. :-)